src/composables/useGetParticipants.js

```diff
@@ -45,12 +56,32 @@ export function useGetParticipants(isActive = ref(true), isTopBar = true) {
 		subscribe('guest-promoted', onJoinedConversation)
 	}
 
+	const handleUsersUpdated = async ([users]) => {
+		const sessionStore = useSessionStore()
```

Comment:
`useSessionStore` is a composable and in theory should be only called in the setup context: in the component's `setup` or in other composables.

It still works here because Pinia tries to simplify it by storing an active pinia instance internally. It works fine while:
- This code is executed later than the pinia instance is created and activated in the Vue app
- There is only one pinia
  - We cannot guarantee it. Any library may create a new Pinia instance

To avoid possible problems here, we should explicitly provide the pinia instance:
- Create a new root module for pinia like we have for router/vuex and export created pinia
- Use this pinia instance in pinia plugin
- Pass it to the composable

And we should make sure that these stores don't use any composables

Reply:
In general, I don't see at the moment why sessions are stored in the Pinia store (UI global reactive state).

This store is never used as a state (this state is not used in UI/computeds, anything reactivity or Vue related). It is only used as an internal variable, and a store is used as a module with functions.

But fine by me, as removing it is a large change.

Reply:
Will move `const sessionStore = useSessionStore()` up to setup context of `useGetParticipants`

I would keep it as all other stores, cause it makes easier at the moment to use as a stepstone for future changes (e.g. if we'll move call-related logic from Vuex participantsStore to here: `peers` state and `phones` state is also updated from signaling messages (see `processTransientCallStatus`)

Reply:
<img width="944" alt="image" src="https://github.com/user-attachments/assets/519da3db-8480-4317-b0da-2903e96dfb73" />